ush/templates/FV3LAM_wflow.xml

```diff
@@ -403,19 +417,111 @@ MODULES_RUN_TASK_FP script.
     </dependency>
 
   </task>
+
+{%- if do_dacycle %}
+<!--
+************************************************************************
+************************************************************************
+-->
+  <task name="&ANAL_GSI_INPUT_TN;" cycledefs="forecast" maxtries="2">
```

Comment:
The maxtries tag should be configurable like the preexisting tasks here and in the next task.

Reply:
Added the maxtries tag for anal_gsi runs.